shell/components/Wizard.vue

```diff
@@ -4,6 +4,7 @@ import AsyncButton from '@shell/components/AsyncButton';
 import { Banner } from '@components/Banner';
 import Loading from '@shell/components/Loading';
 import { stringify } from '@shell/utils/error';
+import LazyImage from '@shell/components/LazyImage';
```

Comment:
I can see that it was present as markup and not imported as a component. How come you noticed it now and why was it not failing? 🤔

Reply:
That's a good callout - this change resolves a warning that was printed to the console

```
[Vue warn]: Failed to resolve component: LazyImage
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement. 
  at <Wizard key=0 ref="Wizard" header-mode="create"  ... > 
  at <CruResource key=1 done-route="c-cluster-product-resource" mode="create"  ... > 
  at <CruGitRepo key=3 ref="comp" value= 
Object { type: "fleet.cattle.io.gitrepo", metadata: {…}, __clone: true, spec: {…}, … }
  ... > 
  at <AsyncComponentWrapper key=3 ref="comp" value= 
Object { type: "fleet.cattle.io.gitrepo", metadata: {…}, __clone: true, spec: {…}, … }
  ... > 
  at <Index component-testid="cluster-manager-create" class="outlet" > 
  at <ClusterResourceCreate class="outlet" onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< 
Proxy { <target>: {…}, <handler>: {…} }
 > > 
  at <RouterView key="/c/_/fleet/fleet.cattle.io.gitrepo/create" class="outlet" > 
  at <Default onVnodeUnmounted=fn<onVnodeUnmounted> ref=Ref< 
Proxy { <target>: {…}, <handler>: {…} }
 > > 
  at <RouterView key=1 > 
  at <App>
```

I've updated the description to include this detail.

Reply:
Wow, should we not make it fail in these cases instead of returning a warning? At least from the build.